bpmn-model/src/test/java/io/camunda/zeebe/model/bpmn/util/time/RepeatingIntervalTest.java

```diff
@@ -198,4 +198,22 @@ public void shouldCalculateDueDate() {
     // then
     assertThat(newDueDate).isEqualTo(expected);
   }
+
+  @Test
+  public void shouldNotBeLessThanCurrentTime() {
```

Comment:
👍 I like the idea of the test case, but it's testing an implementation detail: the interval's toEpochMilis is used to schedule the next timer.

We need a test that makes sure the original behavior is not seen again: the timer shouldn't be scheduled to trigger multiple times if the start time is in the past. 

❌ We can keep the test, but we need an additional test that verifies the behavior: if a repeating timer is defined with a start time in the past, then it should trigger only once and reschedule the following triggers in the future.